### ISSUE TITLE
Running HTTP tests without login when you have more than 1 local DB fails

### ISSUE THREAD

**Issue by Yajo** [odoo/odoo#61472]
Impacted versions: 12 for sure. Others not tested.

Steps to reproduce:

1. Create a database
1. Create another
1. Create an `HttpCase`
1. Call `browser_js(..., login=None)` to test a website tour.
1. Test that module.

Current behavior: The test fails because chrome goes to `/web/database/selector`

Expected behavior: The test case installs a session cookie that indicates what DB is the visitor using, and the visitor opens directly the desired url in the current database.

@CarlosRoca13

**Comment by Yenthe666** [comment]
Hmm I'm not sure if I agree here. I guess you should use a db-filter or some equivalent? There's more cases like this where it fails and redirects to `/web/database/selector` because of not having the session-awareness.
@pedrobaeza can I ask your opinion please? :)

**Comment by pedrobaeza** [comment]
@Yajo I think this behavior is a side effect of not having login, that as a side effect works with no multi-DB, but not the contrary. Or documentation says about using login=None?

**Comment by Yajo** [comment]
Yes, `login=None` is a supported feature. It's also used in Odoo:

https://github.com/odoo/odoo/blob/338f8d843832c339c52c00c7f4c41b5ab2d696ed/addons/website/tests/test_website_reset_password.py#L59

If you have 2 DB when running that test, it'll fail, because it doesn't expect to land in `/web/database/manager`: https://github.com/odoo/odoo/blob/338f8d843832c339c52c00c7f4c41b5ab2d696ed/addons/website/static/tests/tours/reset_password.js#L7-L30

Remember that in `/web/database/manager` controller, no JS assets are loaded, so the tour won't even exist. Thus, it wouldn't even make sense for any tour to land in the database manager.

**Comment by yelizariev** [comment]
That's why I always run tests with ``--db-filter=.* -d devel``
